sphinxcontrib/openapi/openapi30.py

```diff
@@ -229,10 +237,11 @@ def _httpresource(endpoint, method, properties, render_examples):
     # https://github.com/OAI/OpenAPI-Specification/blob/3.0.2/versions/3.0.0.md#operation-object
     parameters = properties.get('parameters', [])
     responses = properties['responses']
-    indent = '   '
+    query_param_examples = {}
+    indent = '    '
```

Comment:
Again does the indent _have_ to change? If not, why do it?

Reply:
This a matter of homogeneity between Sphinx and Python source code.